projects/plugins/jetpack/extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -185,3 +205,4 @@ private static function is_squareish_layout( $attr ) {
 }
 
 Tiled_Gallery::register();
+add_action( 'init', array( 'Automattic\Jetpack\Extensions\Tiled_Gallery', 'inject_javascript_options' ) );
```

Comment:
This can be easier to read using the `__NAMESPACE__` magic constant.

```suggestion
add_action( 'init', array( __NAMESPACE__ . '\Tiled_Gallery', 'inject_javascript_options' ) );
```